dashboard/test/controllers/scripts_controller_test.rb

```diff
@@ -1191,6 +1191,19 @@ class ScriptsControllerTest < ActionController::TestCase
     assert_equal ['All Resources', 'All Standards'], response_body.map {|r| r['name']}
   end
 
+  test "get_unit bypasses cache for edit route" do
+    Rails.application.config.stubs(:levelbuilder_mode).returns true
+    sign_in(@levelbuilder)
+
+    Script.expects(:get_from_cache).never
+    Script.expects(:get_without_cache).returns(@migrated_unit).once
```

Comment:
if possible, it would be nice to check on this line that the `with_associated_models` param is falsy. that seems like more at the center of why you are adding this test than whether the cache method is called or not.